src/TableTransforms.jl

```diff
@@ -38,6 +38,7 @@ export
   Coerce,
   RowTable,
   ColTable,
+  OneHotEncoding,
```

Comment:
I wonder if we could adopt the shorter name OneHot? It should be clear and more concise. What do you think?

Reply:
I think that's a good name.